app/assets/javascripts/taxonomy_edit.js

```diff
@@ -8,7 +8,7 @@ function ignore_checked(item){
   var current_select = $(item).closest('.tab-pane').find('select');
 
   if ($(item).is(':checked')) {
-     current_select.attr('disabled', 'disabled');
+     current_select.prop('disabled', true);
   } else {
      current_select.removeAttr('disabled');
```

Comment:
This should also use `prop`. There are also a couple more occurrences of `removeAttr` for properties such as `readonly` and `selected`, please fix them as well.